app/service/deed_api.py

```diff
@@ -42,7 +43,8 @@ def get_address(address_json):
 def get_borrowers_json(ids):
     borrowers = []
     for borrower_id in ids:
-        response = requests.get(DEED_API_BASE_HOST + '/borrower/' + str(borrower_id))
+        response = requests.get
+        (DEED_API_BASE_HOST + '/borrower/' + str(borrower_id))
```

Comment:
I would suggest 

```
response = requests.get(DEED_API_BASE_HOST +
                        '/borrower/' + str(borrower_id))
```
the indentation in python helps us understand ownership, if this is not indented it looks to a developer like they are 2 separate lines of code.

Feel free to disagree